### ISSUE TITLE
Casting allowed for unpivot operator?

### ISSUE THREAD

**Issue by antonio-olleros** [sdmx-twg/vtl#394]
### Issue Description
According to VTL Manual row 7183, for unpitvo operator: _All the measures of op must be defined on the same Value Domain._
The question is: What happens if we have two measures with different types, but for which implicit casting is allowed (E.g., number and integer)?

### Proposed Solution
Whatever solution should be documented in the manual

**Comment by vpinna80** [comment]
I tend to favor raising an error in this case to avoid surprising results, like numbers converted to strings or vice versa

**Comment by antonio-olleros** [comment]
I tend to agree, but only because of the implicit conversion to string... Actually, I was thinking that maybe what we should do is to forbid the implicit conversion to string, because it can only cause silent errors... If you want to do a cast, you just need to do it explictly, except when it is a case from a subtype to the supertype. What do you think about this option? I can open an issue

**Comment by javihern98** [comment]
An example is still needed in the new documentation as per https://github.com/sdmx-twg/vtl/blob/f306668e7301e8ec8fab3a49d96546543352ab0a/meetings/2025/June_Salamanca/june_20_salamanca.md?plain=1#L61

**Comment by linardian** [comment]
Hello, @javihern98.
I added example #2 in the docs (see version 2.2, while 2.1 has just 1 example).
I think this would be enough. Thoughts?

**Comment by linardian** [comment]
v2.2/docs/reference_manual/operators/Clause operators/Unpivoting/examples
